EpsilonDashboardSetup.sh

```diff
@@ -0,0 +1,32 @@
+#!usr/bin/env bash
+
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | sudo tee /etc/apt/sources.list.d/rabbitmq.list && sudo apt-get update && sudo apt-get install rabbitmq-server
+sudo apt-get install cmake libboost-dev openssl libssl-dev libblkid-dev e2fslibs-dev libboost-all-dev libaudit-dev software-properties-common build-essential mesa-common-dev libgl1-mesa-dev
+
+cd /tmp/
+if [ -e /usr/local/librabbitmq.*]
```

Comment:
Need a space between `*` and `]`

Reply:
The wildcard didn't work for me, I think I remember us looking at this:
https://stackoverflow.com/questions/6363441/check-if-a-file-exists-with-wildcard-in-shell-script